lib/bcx.rb

```diff
@@ -16,6 +16,7 @@ module Resources
     autoload :Person, 'bcx/resources/person'
     autoload :Access, 'bcx/resources/access'
     autoload :Authorization, 'bcx/resources/authorization'
+    autoload :Comment, 'bcx/resources/comment'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.